luigi/parameter.py

```diff
@@ -140,6 +140,10 @@ def __init__(self, default=_no_value, is_global=False, significant=True, descrip
                                 ``positional=False`` for abstract base classes and similar cases.
         :param bool always_in_help: For the --help option in the command line
                                     parsing. Set true to always show in --help.
+        :param function(iterable[A]) -> A batch_method: Method to combine an iterable of parsed
+                                                        parameter values into a single value. Used
+                                                        when receiving batched parameter lists from
+                                                        the scheduler.
```

Comment:
I just created #1814 which is going to be able to help you.

As for referencing to a section in the documentation. See this example:

 * Place where you refer: https://github.com/spotify/luigi/blob/298a0d13e812f5613e39d86a24f51ac9348b66be/luigi/task.py#L591
 * Place where you refer to: https://raw.githubusercontent.com/spotify/luigi/298a0d13e812f5613e39d86a24f51ac9348b66be/doc/configuration.rst (the line containing `.. _ConfigClasses:`)